modules/nw-ingress-setting-thread-count.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-setting-thread-count_{context}"]
+= Setting Ingress Controller thread count
+A cluster administrator can set the thread count to increase the amount of incoming connections a cluster can handle. You can patch an existing Ingress Controller to increase the amount of threads.
+.Prerequisites
```

Comment:
Need a blank line between lines 7 & 8 so that it renders `.Prerequisites` on a new line instead of appending it to the paragraph ([see output](https://deploy-preview-32853--osdocs.netlify.app/openshift-enterprise/latest/networking/ingress-operator?utm_source=github&utm_campaign=bot_dp#nw-ingress-setting-thread-count_configuring-ingress))